src/status_im2/contexts/chat/lightbox/text_sheet/style.cljs

```diff
@@ -13,10 +13,11 @@
     :left     0
     :right    0}))
 
-(def text-style
+(defn text-style
+  [expanding-message?]
   {:color             colors/white
    :margin-horizontal 20
-   :margin-bottom     constants/text-margin
+   :align-items       (if expanding-message? nil :center)
```

Comment:
btw recently found out GH has a suggestion feature from which small fixes like this can be committed quickly
```suggestion
   :align-items       (when-not expanding-message? :center)
```